db/db-config.js

```diff
@@ -0,0 +1,28 @@
+var pg = require('pg');
```

Comment:
Was this part of the Heroku setup tutorial? If not, again really well done with setting up the module for use elsewhere in the app.